src/app/resperson/index.njk

```diff
@@ -0,0 +1,88 @@
+{% extends "base-template.njk" %}
+
+{% import "common/templates/includes/macros.njk" as m %}
+{% import "common/templates/includes/pagination.njk" as pagination %}
+
+{% block pageTitle %}
+	Manage responsible person | Submit a General Aviation Report (GAR)
+{% endblock %}
+
+{% set active = '/resperson' %}
+
+{% block header %}
+  {% include "header-user.njk" %}
+{% endblock %}
+
+{% block beforeContent %}
+  {% include "phase.njk" %}
+{% endblock %}
+
+{% block content %}
+<script type="text/javascript" src="/javascripts/resperson_utils.js"></script>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      {% if errors %}
+        {% include "common/templates/includes/errors.njk" %}
+      {% endif %}
+
+      {% if successMsg %}
+        {% include "common/templates/includes/success-action.njk" %}
+      {% endif %}
+
+      <h1 class="govuk-heading-xl">{{ __('heading_responsible_person') }}</h1>
+      <p class="govuk-body-l">{{__('res_person_caption')}}</p>
+      <p class="govuk-body-m">Search for responsible person </p>
+      <input class="govuk-input govuk-input--width-20" id="resperson-search" name="resperson-search" type="text" placeholder="Search" style="margin-bottom: 10px">
+    </div>
+    
+    <div class="govuk-grid-column-full">
+      <p class="govuk-body-s">{{__('person_sort')}}</p>
+      <form id="responsiblePersonId" action="/resperson" method="post">
+        <table class="govuk-table" id="res_person_table">
+          <thead class="govuk-table__head">
+            <tr class="govuk-table__row">
+              <th class="govuk-table__header" id="surname" scope="col">{{__('field_surname')}}</th>
+              <th class="govuk-table__header" id="given_name" scope="col">{{__('field_given_name')}}</th>
+              <th class="govuk-table__header" id="email" scope="col">{{__('field_email')}}</th>
+              <th class="govuk-table__header" scope="col">{{__('table_action')}}</th>
+            </tr>
+          </thead>
+          <tbody class="govuk-table__body" id="resPerson_table_row">
+            {% if resPersons | length %}
+              {% for resPerson in resPersons %}
+                  <tr class="govuk-table__row">
+                    <td class="govuk-table__cell">
+                      <button id="edit-edit_reponsiblePerson-{{ loop.index }}" class="govuk-link btn-link" 
```

Comment:
`Optional`: Larger design change so I understand if this should be consider at a late date or not at all.
* Instead of `editResponsiblePerson` and `value="{{resPerson.responsiblePersonId}}` or `name="deleteResponsiblePerson" value="{{resPerson.responsiblePersonId}}"`, `formaction` property on the button could be used instead to direct the edit or delete actions e.g.
```njk
{# instead of the below #}
<button id="edit-edit_reponsiblePerson-{{ loop.index }}" class="govuk-link btn-link" 
                      title="Edit details for {{ resPerson.responsibleGivenName }} {{ resPerson.responsibleSurname }}" 
                      name="editResponsiblePerson" value="{{resPerson.responsiblePersonId}}">{{resPerson.responsibleGivenName}}
                      </button>
{# This can be done #}
<button id="edit-edit_reponsiblePerson-{{ loop.index }}" class="govuk-link btn-link" 
                      title="Edit details for {{ resPerson.responsibleGivenName }} {{ resPerson.responsibleSurname }}" 
                     formaction="resperson/edit" formmethod="/get" name="editResponsiblePerson" value="{{resPerson.responsiblePersonId}}">{{resPerson.responsibleGivenName}}
                      </button>
```
This removes the need for `src/app/resperson/post.controller.js` routing.

Reply:
updated and removed the post controller

Reply:
tests good.